hmda/src/test/scala/hmda/validation/rules/lar/validity/2018/V699Spec.scala

```diff
@@ -1,29 +1,11 @@
-package hmda.validation.rules.lar.validity.eighteen
+package hmda.validation.rules.lar.validity_2018
 
 import hmda.model.filing.lar.LoanApplicationRegister
 import hmda.validation.rules.EditCheck
 import hmda.validation.rules.lar.LarEditCheckSpec
 import hmda.model.filing.lar.LarGenerators._
-import hmda.model.filing.lar.enums.{
-  Accept,
-  ApproveEligible,
-  ApproveIneligible,
-  Caution,
-  Eligible,
-  EmptyAUSResultValue,
-  Error,
-  Incomplete,
-  Ineligible,
-  Invalid,
-  OtherAUS,
-  OtherAutomatedUnderwritingResult,
-  OutOfScope,
-  Refer,
-  ReferEligilbe,
-  ReferIneligible,
-  ReferWithCaution,
-  UnableToDetermineOrUnknown
-}
+import hmda.model.filing.lar.enums.{Accept, ApproveEligible, ApproveIneligible, Caution, Eligible, EmptyAUSResultValue, Error, Incomplete, Ineligible, Invalid, OtherAUS, OtherAutomatedUnderwritingResult, OutOfScope, Refer, ReferEligilbe, ReferIneligible, ReferWithCaution, UnableToDetermineOrUnknown}
```

Comment:
one long line seems like it would be kind of hard to read, or should we just do `._`?